server/src/gateways/judges.gateway.ts

```diff
@@ -53,13 +56,24 @@ export class JudgesGateway implements FightEndConditionFulfilledObserver {
     }
 
     const fight: FightImpl = this.fightsService.getFight(fightId);
+    const role = fight.getJudgeRole(judgeId);
+    const connectedJudges = fight.getConnectedJudgesStatus();
+    connectedJudges.delete(role);
```

Comment:
Myślę, że nawet nie trzeba usuwać naszej roli z tego zbioru - frontend i tak wie kim jesteśmy dzięki polu "role" w odpowiedzi, a dzięki temu kod będzie prostszy i frontend będzie mógł łatwiej sprawdzić ile jeszcze osób nam brakuje i jakie są brakujące role.

Reply:
jeżeli nie zdecydujemy się na sendToAllJudges, to trzeba usunąć. Wysyłanie więcej danych niż trzeba też nie wydaje mi się dobrym rozwiązaniem.

Reply:
To prawda, wszystko zależy od decyzji z `sendToAllJudges` (pisałem komentarz pod kątem sytuacji, w której implementacja opiera się na `sendToAllJudges`)